src/spikeinterface/extractors/neoextractors/neuralynx.py

```diff
@@ -27,22 +29,44 @@ class NeuralynxRecordingExtractor(NeoBaseRecordingExtractor):
     exlude_filename: list[str], default: None
         List of filename to exclude from the loading.
         For example, use `exclude_filename=["events.nev"]` to skip loading the event file.
+    strict_gap_mode: bool, default: False
+        See neo documentation.
+        Detect gaps using strict mode or not.
+          * strict_gap_mode = True then a gap is consider when timstamp difference between two
+            consequtive data packet is more than one sample interval.
+          * strict_gap_mode = False then a gap has an increased tolerance. Some new system with different clock need this option
```

Comment:
```suggestion
          * strict_gap_mode = False then a gap has an increased tolerance. Some new systems with different clocks need this option
```

It would be nice to know which systems, but I know this isn't super clear.